service/accounts/openstack_manager.py

```diff
@@ -28,76 +21,21 @@
 
 from atmosphere import settings
 
-from core.models import AtmosphereUser as User
-from core.ldap import get_uid_number
 from core.models.identity import Identity
 
 from service.accounts.base import BaseAccountDriver
+from service.networking import get_topology_cls, ExternalRouter, ExternalNetwork
 
 from atmosphere.settings.secrets import SECRET_SEED
-from atmosphere.settings import DEFAULT_PASSWORD_LOOKUP, DEFAULT_PASSWORD_UPDATE
-
-def get_unique_id(userid):
```

Comment:
This is deleted def is referenced [here](https://github.com/steve-gregory/atmosphere/blob/1d6053d1e7db08cf5002e6331ad2465518aaccd7/service/accounts/openstack_manager.py#L422)